dali/kernels/imgproc/geom/remap_npp.h

```diff
@@ -227,7 +213,7 @@ struct NppRemapKernel : public RemapKernel<Backend, T> {
   }
 
 
-  NppStreamContext npp_ctx_{};
+  NppStreamContext npp_ctx_;
```

Comment:
Why? This is quite risky and offers little, if any, performance benefits.

Reply:
It has hidden a problem with the call ``CUDA_CALL(cudaStreamGetFlags(npp_ctx_.hStream, &npp_ctx_.nStreamFlags));`` accessing hStream that has not been set. So I left a version that actually reported problems that stream is not set when it is not set.